indico/modules/events/papers/templates/management/cfp_actions/_base.html

```diff
@@ -0,0 +1,27 @@
+{% extends '_module_actions_base.html' %}
+
+{% block icon_classes %}icon-file{% endblock %}
+
+{% block buttons %}
+    {% set title -%}
+        {%- if cfp.start_dt -%}
+            {%- trans %}Reschedule call for papers{% endtrans -%}
+        {%- else -%}
+            {%- trans %}Schedule call for papers{% endtrans -%}
+        {%- endif -%}
+    {%- endset -%}
+    <div class="group">
```

Comment:
but then it looks like this
![gap](https://cloud.githubusercontent.com/assets/9071624/21306689/9a6e76e6-c5d2-11e6-9efc-83f8f1951aa9.png)
compared to this
![ggap](https://cloud.githubusercontent.com/assets/9071624/21306706/af7e9188-c5d2-11e6-9121-b914982deb67.png)
So I guess we want to keep the gap there